.github/workflows/backend.yml

```diff
@@ -95,7 +95,7 @@ jobs:
           kafka: true
           snuba: true
           symbolicator: true
-          peanutbutter: true
```

Comment:
if we're not putting this in production can we delete all the code for it here and archive the service repo? this'll save me some work I need to do to port it to artifact registry as well

Reply:
I think that's probably a good idea. And it's not like we can't put it in again if it happens at some point. I'll adapt this PR.